Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
+         if File.directory?(f)
```

Comment:
It does. I tried indenting it as you'd expect it to be indented, and I got a stream of ` expected end ` failures. I'll take another look shortly... Working on it at 3am probably made me less tolerant towards failures than I normally am :smile_cat: